test/conformance/behaviors/README.md

```diff
@@ -0,0 +1,49 @@
+# Behaviors Defining Conformance
+
+The conformance program is intended to answer the question "What is
+Kubernetes?". That is, what features, functions, and APIs are needed in order to
+call something "Kubernetes". Since v1.9, this has been defined as passing a
+specific set of e2e tests.  As part of the [conformance behavior KEP](https://git.k8s.io/enhancements/keps/sig-architecture/20190412-conformance-behaviors.md),
+this instead is moved to an explicit list of "behaviors", which are captured in
+this directory tree. The e2e tests are used to validate whether specific
+behaviors are met, but the definition of conformance is based upon that list of
+approved behaviors. That begs the question, "what is a behavior?".
+
+In behavior driven development, it is sometimes defined as the "business logic"
+expected from the software. That is, it is a sequence of reactions to some
+stimulus: an API call, a component failure, a network request on the data plane,
+or some other action. We can classify these reactions into a few different types
+in Kubernetes:
+
+1. Transient runtime / communication state
+1. Cluster state changes observable via the API
+1. Cluster state changes observable via the data plane
+
+Another way to think about this is that a behavior is the combination of the
+question and answer to "What happens when...".
+
+A behavior will consist of:
+
+* A description of the initial conditions and stimulus
+* A description of the resulting reactions
+* In some cases, a sequencing (ordering) of those reactions
```

Comment:
This seems to conflict with the point 

> Ordered sequencing of the reactions should be avoided unless absolutely
   necessary.

What defines "absolutely necessary"? Perhaps some examples would help. There's already a TODO at the bottom so feel free to include with a follow up PR

Reply:
Added some color for this.